askama_shared/src/escaping.rs

```diff
@@ -55,26 +55,25 @@ fn escapable(b: u8) -> bool {
 }
 
 pub fn escape(s: String) -> String {
-    let mut found = Vec::new();
+    let mut found = 0;
+    let mut flag = true;
```

Comment:
I think we can make `found` an `Option<usize>` here, instead of having the separate `flag` variable?

Reply:
It's so cool, :smile: thanks 
```
  Running target/release/deps/all-7a1802b78ef1756b
Big table               time:   [846.27 us 847.09 us 848.03 us]
                        change: [-1.3469% -0.7382% -0.1178%] (p = 0.02 < 0.05)
                        Change within noise threshold.
Found 11 outliers among 100 measurements (11.00%)
  2 (2.00%) high mild
  9 (9.00%) high severe

Teams                   time:   [1.1615 us 1.1647 us 1.1685 us]
                        change: [-1.5642% -0.7306% +0.0928%] (p = 0.09 > 0.05)
                        No change in performance detected.
Found 18 outliers among 100 measurements (18.00%)
  1 (1.00%) high mild
  17 (17.00%) high severe

Escaping                time:   [6.1220 us 6.1350 us 6.1507 us]
                        change: [-0.2161% +0.5854% +1.4861%] (p = 0.18 > 0.05)
                        No change in performance detected.
Found 19 outliers among 100 measurements (19.00%)
  4 (4.00%) high mild
  15 (15.00%) high severe
```
Now I do the commit. thanks